tests/dummy/mirage/factories/github-user.js

```diff
@@ -7,7 +7,8 @@ export default Factory.extend({
   name: function(i) {
     return `User ${i}`;
   },
-  type: 'User',
+  // needs to be the actual related model name for Ember Data reasons? 🤔
+  type: 'github-user',
```

Comment:
there is something funky happening here with how the mirage record is getting serialized which required this to get the tests to behave.  Trying things in a real app I'm not seeing a similar issue.